apps/chronicles/navigation/SettingsScreen.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { SettingsScreen, SettingsScreenConfiguration } from "@equinor/mad-core";
+
+export const SampleSettingsScreen = () => {
+    const myConfig: SettingsScreenConfiguration = [
+        {
+            items: [
+                {
+                    name: "navigation",
+                    title: "navigation",
+                    onPress: () => undefined,
+                    iconName: "abacus",
+                },
+                {
+                    name: "button",
+                    title: "Button",
+                    onPress: () => undefined,
+                    iconName: "abacus",
+                    color: "primary",
+                },
+                {
+                    name: "switch",
+                    title: "Test",
+                    onChange: () => undefined,
+                    isActive: true,
+                    iconName: "abacus",
+                },
+            ],
+        },
+    ];
```

Comment:
Is this a template? Should maybe clarify it, e.g. rename to "templateConfig" or something

Reply:
Fixed